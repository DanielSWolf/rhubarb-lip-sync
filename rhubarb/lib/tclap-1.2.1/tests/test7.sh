#!/bin/sh

# success
../examples/test2 -i 10 -s hello goodbye -hABC > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test7.out; then
	exit 0
else
	exit 1
fi

