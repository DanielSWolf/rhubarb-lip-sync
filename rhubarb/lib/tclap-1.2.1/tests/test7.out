
USAGE: 

   ../examples/test2  [-f <float>] -i <int> -s <string> [-A] [-C] [-B] [--]
                      [--version] [-h] <string> <string> ...


Where: 

   -f <float>,  --floatTest <float>
     float test

   -i <int>,  --intTest <int>
     (required)  integer test

   -s <string>,  --stringTest <string>
     (required)  string test

   -A,  --existTestA
     tests for the existence of A

   -C,  --existTestC
     tests for the existence of C

   -B,  --existTestB
     tests for the existence of B

   --,  --ignore_rest
     Ignores the rest of the labeled arguments following this flag.

   --version
     Displays version information and exits.

   -h,  --help
     Displays usage information and exits.

   <string>
     (required)  unlabeld test

   <string>  (accepted multiple times)
     file names


   this is a message

